src/rundeck/values.yaml

```diff
@@ -0,0 +1,115 @@
+# Default values for rundeck.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: deepakr11/rundeck
```

Comment:
I don't believe you can commit to updating your own docker image in lock step with the community. It'd be better to override the pod spec command line to do things before calling the original entrypoint.

For the values.yaml, use the community image as default. For your own usage, use the helm --set command for your own customizations.

Mount a custom setup script via a configmap. Then override the original entrypoint.  Make it all configurable via values.yaml.